scripts/trlite

```diff
@@ -281,11 +281,12 @@ if [ "$RUN" == "all" -o "$RUN" == "1" ]; then
     echo "console.log(3.14159);" >> $TMPFILE
     try_command "modules" make $VERBOSE JS=$TMPFILE ROM=256
 
+    # FIXME: Disabled until iot-c gets rebased to work with latest Zephyr
```

Comment:
This is not needed if you pull latest iotivity-constraint, which has the zephyr fix.